param/low_level_motion_0.05.yaml

```diff
@@ -40,3 +40,5 @@ roll_max_rate     : 1.0
 yaw_max           : 3.0
 yaw_min           : -3.0
 yaw_max_rate      : 600.0
+
+update_frequency: 60.0
```

Comment:
It was previously limited by the transform frequency.  Now it's not.  robot_localization publishes transforms at 30Hz, as does fc_comms, so I could move this down to 30Hz, but we don't have to